include/fmt/base.h

```diff
@@ -1475,6 +1502,19 @@ template <typename Context> struct arg_mapper {
   FMT_MAP_API auto map(double val) -> double { return val; }
   FMT_MAP_API auto map(long double val) -> long double { return val; }
 
+#if FMT_USE_BITINT
```

Comment:
Let's remove this check and make sure that the overloads below are disabled if _BitInt is not available.